### PR TITLE
exiv2: 0.27.1 -> 0.27.2

### LINKED ISSUE #67194: Vulnerability roundup 74: exiv2-0.27.1: 1 advisory
[search](https://search.nix.gsc.io/?q=exiv2&i=fosho&repos=nixos-nixpkgs), [files](https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=exiv2+in%3Apath&type=Code)

* [ ] [CVE-2019-14982](https://nvd.nist.gov/vuln/detail/CVE-2019-14982) (nixos-unstable)

Scanned versions: nixos-unstable: 1412af4b2cf. May contain false positives.

### PR DESCRIPTION
**Pull request by mmahut**
###### Motivation for this change

Fixes #67194

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build empathy

**Comment by worldofpeace** [comment]
I have
```
    # Fix tests on Aarch64
```
in the expression. Can we check if that's fixed?

**Comment by mmahut** [comment]
>  Can we check if that's fixed?

It looks like it does, yay.

**Comment by mmahut** [comment]
Did a full rebuild over night, no new broken packages.

**Comment by worldofpeace** [comment]
> Did a full rebuild over night, no new broken packages.

Excellent, thanks :+1:

File: pkgs/development/libraries/exiv2/default.nix [hunk 1]
```diff
@@ -58,8 +53,6 @@ stdenv.mkDerivation rec {
     "doc"
   ];
 
-  doCheck = stdenv.isLinux;
```
Comment (worldofpeace) [inline pkgs/development/libraries/exiv2/default.nix]: `doCheck` is default false so this disabled them.

Comment (mmahut) [reply pkgs/development/libraries/exiv2/default.nix]: Oh :(

**Pull request merged**